src/components/data/DataTable.js

```diff
@@ -346,6 +372,12 @@ const DataTable = props => {
               }
             }
 
+            const filterBreadCrumb = h(div, { style: { display: 'flex', overflow: 'hidden' } }, [h(TextCell, { style: { fontWeight: 400, ...Style.noWrapEllipsis } }, [`filtered by: ${filterColTerm}`]), h(Clickable, {
```

Comment:
I suggest changing `h(TextCell` to `h(TooltipCell` to get a nice tooltip for long filter values, like:
![Screenshot 3-14-2023 at 05 32 PM](https://user-images.githubusercontent.com/6041577/225142664-0bf8618e-8c67-42d0-a087-7dc4dce22702.png)

Reply:
Done!  Looks great!